### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by dogukanteber**
This PR adds the following files:

* README.md
* GOVERNANCE.md
* CONTRIBUTING.md
* CODE_OF_CONDUCT.md
* MAINTAINERS

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# auth-gateway Governance
+
+This document defines project governance for the project.
+
+## Voting
+
+todo
```
Comment (dogukanteber) [inline GOVERNANCE.md:7]: Since currently there are only two maintainers, I am not sure how to fill this section

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Changes in Maintainership
+
+Contributors who are interested in becoming a maintainer, if performing relevant responsibilities, should discuss their interest with the existing maintainers. New maintainers must be nominated by an existing maintainer and must be elected by 2/3 majority vote.
+
+We do not expect anyone to make a permanent commitment to be a auth-gateway maintainer forever. After all, circumstances change, people get new jobs, new interests, and may not be able to continue contributing to the project. At the same time, we need to keep the list of maintainers current in order to have effective governance. People may be removed from the current list of maintainers via one of the following ways:
+
+
+* They can resign
+* If they stop contributing to the project for a period of 6 months or more
+* By a 2/3 majority vote from active maintainers
```
Comment (dogukanteber) [inline GOVERNANCE.md:18]: The same comment I wrote above applies here too. I wrote this as a placeholder. It is open to change

File: README.md [hunk 3]
```diff
@@ -1 +1,135 @@
-# auth-gateway
\ No newline at end of file
+<p align="center"><img src="images/logo.png" alt="auth-gateway Logo"></p>
+
+[![CI](https://github.com/cortexproject/auth-gateway/workflows/ci/badge.svg)](https://github.com/cortexproject/auth-gateway/actions)
+[![GoDoc](https://godoc.org/github.com/cortexproject/auth-gateway?status.svg)](https://godoc.org/github.com/cortexproject/auth-gateway)
+<a href="https://goreportcard.com/report/github.com/cortexproject/auth-gateway"><img src="https://goreportcard.com/badge/github.com/cortexproject/auth-gateway" alt="Go Report Card" /></a>
+<a href="https://cloud-native.slack.com/messages/cortex/"><img src="https://img.shields.io/badge/join%20slack-%23cortex-brightgreen.svg" alt="Slack" /></a>
+<!-- <a href="https://bestpractices.coreinfrastructure.org/projects/6681"><img src="https://bestpractices.coreinfrastructure.org/projects/6681/badge"></a> -->
```
Comment (dogukanteber) [inline README.md:7]: We had to sign in and add `auth-gateway` to this website. However, I was not sure which e-mail address I should use to sign in. I though maybe there is an e-mail dedicated to Cortex so I did not add the project and commented out this line.

File: README.md [hunk 4]
```diff
@@ -1 +1,135 @@
...
...Bq2b5zXoWBaV7XB7b3%2B27tsNK%2BsOuy%2BjM7bdqc26I%2F6U6V%2BBayzqKXeCtq9%2B7vR7fZ49%2B3C%2BdfxavX2tfr8MlJHz91fD093bvNh%2FNb1%2F5WiEFxN%2FlzevJ2%2Fzq666D4eJ2FN%2Fq2Nfp89GLfAG4TGM%2Fr25%2FEI9i%2Bb6Hf2Jh3rtnHdvAT9kRkh9vvzi5en2cXrYGS1PPnKBFqzOetJ6jy6bjemzzLfmFqDsH5r81aE6PP2oM359rSN36tnTR6%2BXbU851f02DhT3pz%2Bnemq3tUFeNZC874ZLc4uwPRShhfO9cVT%2B2kRiB1rBOH5g8M%2F2IPauebVvNHNxY0yusfUrF89dJUzf3Rl27jMPf5XTst1RojF9xaCNYXEevxjGfV1bKnO8RSvCkp1trVlCFwHBqcONzbRvfZcwFnoPsRWfDUmUivOJbk4kdTiZa0bPN40%2FGjSHY%2BDyAknnuDN4ml8eBp%2FhsNWEod79%2F8KoqzPNV7kpoIqSpImG6qStLKU4nJwaaHdRkxoPGcYB4SFRPuPS29jHD%2FD58f0LGB6Zhdwlcnng9qejKSrB0Nn4ICgDBv0IFGlIh27603p2a50zihktX6IjkJ7MLB5JVXKstWJynuDL27ZalunKs%2ByFejVrUD7XNFdDVjp7I1NeO8J7l2a8Aq17DDhtRXUefd0%2FQCsn22IEbONxIqYC4I7t7EQkeG0pvwjRJOP5iGu%2F3RbSvoUysP2jL6jStv%2F%2Fh8Va1DNC%2FPP9qcWg6ci7QRPco%2FusuHJimgtd%2FLqO9N0V650CK%2F2GSjvur0XAurw1E4fKbmcFN3hct7sd6HhzB0QMrRTwrIA6BibJ1Cw%2B9B6JDdK1ziJX%2F8TKFNSYGUqZJlTS2hcrVRpV5GgZZE4Yyld7cutJL7KbJMIs00rWFCqEGF00kUpz2Bj9UGNYZwDSXIhjJXwwLusfD4rnntflSzo9I0qA2%2B1okvZ6IJQCoyILlHi86Xsw8J4cuDOS4ZOLRn8wLSyn2kgxDtxXvwXmNGry2%2FgmfG31dZ3o6SWlexqU8noSAlrkKhk%2BZ4uSWu6y9hT%2FCJdYKoCh%2FoPHERdVDk9G9BmFEyUhAh06EPMi3d9gvcVacM%2B1vRn%2Fwc%3D)
```
Comment (dogukanteber) [inline README.md:23]: I put the link of the diagram in case we want to update it.

File: README.md (hunk 3 above)
Comment (friedrichg) [reply README.md:7]: There is an email for cortex mantainers, I will send you that email, we can use that

**Review by friedrichg — APPROVED** [review]
LGTM, sorry for taking so long. I had many things in play

**Comment by dogukanteber** [comment]
No problem @friedrichg

**Pull request merged**